src/plugins/translation-inject.js

```diff
@@ -17,14 +17,17 @@ module.exports = (opts = {}) => {
       const magicString = new MagicString(code);
       this.parse(code, {
         onComment: (block, text, start, end) => {
-          const match = text.match(/@ngTranslationsInject(.*)/);
+          const match = text.match(/@ngTranslationsInject:?([a-zA-Z]+)?(.*)/);
           if (match) {
             const translations = _.chain(match)
-              .get(1)
+              .get(2)
```

Comment:
will it works if format is not provided ?

Reply:
Yes, format will be `undefined` and by default, format is xml :)

Cf: https://github.com/ovh-ux/component-rollup-config/pull/22/files#diff-cb5d684f7dcca02f073d4210651e473fR72